<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>校园导览</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/img/home.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#homeButton{
	border: none;
	background-color: rgb(240, 240, 240);
	image: url(:/images/img/home.png);
	padding: 10px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>51</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgb(255, 255, 213);
image: url(:/images/img/set.png);
padding: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_home" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>571</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="text_2">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>365</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>终点：</string>
     </property>
    </widget>
    <widget class="QLabel" name="pathText">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>410</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pathText{
	background-color: rgba(221, 245, 245, 0.3);
	border-style: solid;
	border-radius: 7px;
	border-width: 1px;
	border-color: rgb(193, 230, 240);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="site_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/img/position.png);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="map">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>30</y>
       <width>430</width>
       <height>294</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/img/map.png);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="site_1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/img/position.png);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_begin">
     <property name="geometry">
      <rect>
       <x>166</x>
       <y>360</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="site_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/img/position.png);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="site_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/img/position.png);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_end">
     <property name="geometry">
      <rect>
       <x>366</x>
       <y>360</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="text_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>420</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>推荐路径：</string>
     </property>
    </widget>
    <widget class="QLabel" name="text_1">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>365</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>起点：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="site_0">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>270</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/img/position.png);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>text_2</zorder>
    <zorder>site_4</zorder>
    <zorder>map</zorder>
    <zorder>site_1</zorder>
    <zorder>site_3</zorder>
    <zorder>site_2</zorder>
    <zorder>text_3</zorder>
    <zorder>text_1</zorder>
    <zorder>site_0</zorder>
    <zorder>combo_begin</zorder>
    <zorder>combo_end</zorder>
    <zorder>pathText</zorder>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 253, 253);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>widget_home</zorder>
   <zorder>label</zorder>
   <zorder>setButton</zorder>
   <zorder>homeButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
